changelog.d/20768-default_gelf_stream_framing.breaking.md

```diff
@@ -0,0 +1,31 @@
+Now the GELF codec with stream-based sources uses null byte (`\0`) by default as messages delimiter instead of newline (`\n`) character.
```

Comment:
```suggestion
Now the GELF codec with stream-based sources uses null byte (`\0`) by default as messages delimiter instead of newline (`\n`) character. This better matches GELF server behavior.
```